examples/fluids/qfunctions/freestream_bc.h

```diff
@@ -81,6 +81,28 @@ CEED_QFUNCTION_HELPER StateConservative Flux_HLL_fwd(State left, State right, St
   };
 }
 
+CEED_QFUNCTION_HELPER void ComputeHLLSpeeds_Roe(NewtonianIdealGasContext gas, State left, CeedScalar u_left, State right, CeedScalar u_right,
+                                                CeedScalar *s_left, CeedScalar *s_right) {
+  const CeedScalar gamma = HeatCapacityRatio(gas);
+
+  RoeWeights r = RoeSetup(left.U.density, right.U.density);
+  // Speed estimate
+  // Roe average eigenvalues for left and right non-linear waves
+  // Stability requires that these speed estimates are *at least*
+  // as fast as the physical wave speeds.
```

Comment:
Are these comments relevant anymore? Particularly the stability comment since we're using Roe speeds regardless and give justification for that  on line 101.

Reply:
Nah, they can be stripped or reorganized.